features/admin.actions.v1/api/use-get-action-by-id.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import useRequest, {
+    RequestConfigInterface,
+    RequestErrorInterface,
+    RequestResultInterface
+} from "@wso2is/admin.core.v1/hooks/use-request";
+import { store } from "@wso2is/admin.core.v1/store";
+import { HttpMethods } from "@wso2is/core/models";
+import { ActionResponseInterface } from "../models/actions";
+
+/**
+ * Hook to get the action configurations by id.
+ *
+ * @param actionType - Type of the action.
+ * @param actionId - ID of the action.
+ * @returns SWR response object containing the data, error, isLoading, isValidating, mutate.
+ */
+const useGetActionById = <Data = ActionResponseInterface, Error = RequestErrorInterface>
+    (actionType: string, actionId: string): RequestResultInterface<Data, Error> => {
+
+    const requestConfig: RequestConfigInterface = {
+        headers: {
+            Accept: "application/json",
+            "Content-Type": "application/json"
+        },
+        method: HttpMethods.GET,
+        url: `${ store.getState().config.endpoints.actions }/${ actionType }/${ actionId }`
+    };
+
+    const shouldFetch: boolean = actionId !== undefined && actionId !== null;
```

Comment:
You can use the `isEmpty` function from lodash to validate the string here.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7168/commits/1ba4f62f4c5620a326ae4ee9e34ab74d23d98e0b